src/main/java/com/amazon/ion/impl/macro/EncodingDirectiveReader.kt

```diff
@@ -0,0 +1,145 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.impl.macro
```

Comment:
This should probably be at `com.amazon.ion.impl`.

Reply:
Sure, I can move it.